docs/upgrade-smart-agent.md

```diff
@@ -0,0 +1,210 @@
+# Overview
+
+On February 21, 2020, SignalFx will launch the version of the Smart Agent, version 5.0.
+
+At a high level, to better integrate with Splunk, this new version of the Smart Agent will be signed with Splunk certificates.
+
+SignalFx recommends that you upgrade your Smart Agent as soon as possible.
+
+Current users can continue to run Smart Agent 4.X; however, after April 21, 2020, any new installs or upgrades will require you to change to the new Splunk-signed repositories.
+
+## Upgrade the Smart Agent
+
+Based on how you have installed the agent, review the relevant method to learn how to upgrade the Smart Agent:
+
+### Method 1: APT
+
+1. Delete the existing SignalFx apt key. (You must run this command as root.)
+
+```sh
+apt-key del 5AE495F6
+```
+
+2. Import the new key: (You must run this command as root.)
+
+```sh
+curl https://dl.signalfx.com/splunk-B3CD4420.gpg | apt-key add -
+```
+
+3. Replace /etc/apt/sources.list.d/signalfx-agent.list with the following contents:
+
+```sh
+deb  https://dl.signalfx.com/debs/signalfx-agent/release /
+```
+
+4. Update the package metadata. (You must run this command as root.)
+
+```sh
+apt-get update
+```
+
+5. Install the latest signalfx-agent. (You must run this command as root.)
+
+```sh
+apt-get upgrade signalfx-agent
+```
+
+### Method 2: YUM:
+
+1. Delete the existing SignalFx RPM key. (You must run this command as root.)
+
+```sh
+rpm -e gpg-pubkey-098acf3b-55a5351a
+```
+
+2. Modify /etc/yum.repos.d/signalfx-agent.repo. Specifically: 
+
+ * Change baseurl to https://dl.signalfx.com/rpms/signalfx-agent/release
+
+ * Change gpgkey to https://dl.signalfx.com/splunk-B3CD4420.pub
+
+3. Upgrade to the latest signalfx-agent package. (You must run this command as root.)
+
+```sh
+yum update signalfx-agent
+```
+
+4. You will be prompted to import the key with the fingerprint 58C3 3310 B7A3 54C1 279D  B669 5EFA 01ED B3CD 4420. Press **y** to accept.
+
+### Method 3: Chef, Ansible, Puppet, and Salt
```

Comment:
The cardinal numbering here provides no useful information.  Just make the headers `### Chef, Ansible, Puppet, Salt`, etc.